beta/src/pages/learn/sharing-state-between-components.md

```diff
@@ -73,59 +73,59 @@ h3, p { margin: 5px 0px; }
 
 </Sandpack>
 
-Notice how pressing one panel's button does not affect the other panel--they are independent.
+गौर कीजिये किस तरह एक panel का बटन दबाना दूसरे panel को प्रभावित नहीं करता--दोनों स्वतंत्र हैं।
 
 <DiagramGroup>
 
 <Diagram name="sharing_state_child" height={367} width={477} alt="Diagram showing a tree of three components, one parent labeled Accordion and two children labeled Panel. Both Panel components contain isActive with value false.">
 
-Initially, each `Panel`'s `isActive` state is `false`, so they both appear collapsed
+शुरुआत में दोनों `Panel`s की `isActive` state `false` है, इसलिए दोनों collapsed दिख रहे हैं
```

Comment:
`collapsed` => `कोलैप्स्ड`

Reply:
I have done the above changes, kindly let me know if there are any more to do.